appveyor.yml

```diff
@@ -11,14 +11,16 @@ platform: x64
 environment:
   global:
     TSK_HOME: "C:\\sleuthkit"
-    JDK_HOME: C:\Program Files\Java\jdk1.8.0
+    JDK_HOME: C:\Program Files\BellSoft\LibericaJDK-11-Full
```

Comment:
@gdicristofaro should this be Java 17? It seems like it is mentioning Java 11.

Reply:
Yes, probably.  I'll look into it.